app/components/pipeline/CreateBuildDialog.js

```diff
@@ -77,50 +75,29 @@ class CreateBuildDialog extends React.PureComponent {
               required={true}
             />
 
-            {this.renderMoreOptions()}
+            <CollapsableArea label="Options" maxHeight={250}>
+              <FormTextarea
+                name="build[env]"
+                label="Environment Variables"
+                help="Place each environment variable on a new line, in the format <code>KEY=value</code>"
+                rows={3}
+              />
+              <div style={{ paddingLeft: 18 }}>
+                <input type="hidden" name="build[clean_checkout]" value="0" />
+                <label className="bold"><input name="build[clean_checkout]" type="checkbox" className="absolute" style={{ marginLeft: -18 }} value="1" />Clean checkout</label>
```

Comment:
Oh, good catch! I also discovered that keyboard navigation was broke (you need to set tabIndex to -1 for the collapsed inputs), and some of the other outline and focus styles were broked. Now it should be both accessible to keyboard users and screenreaders, and that outline bug is fixed.